app/components/moderation-list-row/component.js

```diff
@@ -68,5 +84,15 @@ export default Component.extend({
             pending: 'fa-hourglass-o pending',
             rejected: 'fa-times-circle-o rejected',
         };
+
+        this.get('submission.actions').then(this._handleActions.bind(this));
+    },
+
+    _handleActions(actions) {
+        if (actions.length) {
+            this.set('noActions', false);
```

Comment:
Could `noActions` be replaced with a computed property? Maybe depending on `submission.actions.isPending` and `submission.actions.length`?